app/common/lib/suggestion.js

```diff
@@ -239,8 +238,8 @@ const getSortByDomainForSites = (userInputLower, userInputHost) => {
     // any count or frequency calculation.
     // Note that for parsed URLs that are not complete, the pathname contains
     // what the user is entering as the host and the host is null.
-    let host1 = s1.parsedUrl.host || s1.parsedUrl.pathname || s1.location || ''
-    let host2 = s2.parsedUrl.host || s2.parsedUrl.pathname || s2.location || ''
+    let host1 = s1.get('parsedUrl').host || s1.get('parsedUrl').pathname || s1.get('location') || ''
+    let host2 = s2.get('parsedUrl').host || s2.get('parsedUrl').pathname || s2.get('location') || ''
```

Comment:
maybe I'm missing something- shouldn't this be like:
`s1.getIn(['parsedUrl', 'host'])` / `s1.getIn(['parsedUrl', 'pathname']) ?

Reply:
(same with the changes below)

Reply:
Logic was not changed with this PR, so cc @bbondy for this thoughts

Reply:
@NejcZdovc it was changed from mutable to immutable though... which means you have to use `get` in order to read the values. Trying to access like a mutable usually returns `undefined`

Reply:
@bsclifton I see what you mean. Let me check

Reply:
@bsclifton yeah it's working correctly. You can have regular object in immutable object

![image](https://user-images.githubusercontent.com/9574457/30016023-8d132148-9153-11e7-9e75-65da9a282055.png)